kubectl-plugin/test/e2e/kubectl_ray_test.go

```diff
@@ -0,0 +1,68 @@
+package e2e
+
+import (
+	"bytes"
+	"os/exec"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/printers"
+)
+
+func TestKubectlRayGet(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Kubectl Ray Get")
+}
+
+var _ = Describe("Calling ray plugin `get` command", Ordered, func() {
+	It("succeed in getting ray cluster information", func() {
+		cmd := exec.Command("kubectl", "ray", "cluster", "get", "--namespace", "default")
+		output, err := cmd.CombinedOutput()
+
+		expectedOutputTablePrinter := printers.NewTablePrinter(printers.PrintOptions{})
+		expectedTestResultTable := &v1.Table{
+			ColumnDefinitions: []v1.TableColumnDefinition{
+				{Name: "Name", Type: "string"},
+				{Name: "Namespace", Type: "string"},
+				{Name: "Desired Workers", Type: "string"},
+				{Name: "Available Workers", Type: "string"},
+				{Name: "CPUs", Type: "string"},
+				{Name: "GPUs", Type: "string"},
+				{Name: "TPUs", Type: "string"},
+				{Name: "Memory", Type: "string"},
+				{Name: "Age", Type: "string"},
+			},
+		}
+
+		expectedTestResultTable.Rows = append(expectedTestResultTable.Rows, v1.TableRow{
+			Cells: []interface{}{
+				"raycluster-sample",
+				"default",
+				"1",
+				"1",
+				"1",
+				"0",
+				"0",
+				"3Gi",
+				"",
+			},
+		})
+
+		var resbuffer bytes.Buffer
+		bufferr := expectedOutputTablePrinter.PrintObj(expectedTestResultTable, &resbuffer)
+		Expect(bufferr).NotTo(HaveOccurred())
+
+		Expect(err).NotTo(HaveOccurred())
+		Expect(string(output)).To(ContainSubstring(resbuffer.String()))
```

Comment:
not a blocker, but this can probably be a equality check instead of a substring check?

Reply:
unfortunately, the "age" field is not deterministic so a equal check will fail.

Reply:
For some reason this turns out to be false, so currently still trying to figure out what the issue is.

Reply:
Try:
```
Expect(string(output)).To(Equal(resbuffer.String()))
```

Reply:
It won't be equal due to the "age" field right? I did not put an age field in the expected output table since I do not know what the exact age of the cluster will be when the test is run.

Reply:
Ah, I see. Okay let's leave it as a substring check for now